exasol_script_languages_developer_sandbox/lib/asset_id.py

```diff
@@ -0,0 +1,18 @@
+BUCKET_PREFIX = "slc_developer_sandbox"
```

Comment:
could be a class variable if only used within the `AssetId` class

```python3
class AssetId:
   BUCKET_PREFIX = "slc_developer_sandbox"
  

```

Reply:
Done